eng/common/cross/build-rootfs.sh

```diff
@@ -138,6 +138,15 @@ while :; do
             __UbuntuPackages=$(echo ${__UbuntuPackages} | sed 's/ libomp5//')
             unset __LLDB_Package
             ;;
+	ppc64le)
```

Comment:
Changes under `eng/common` should be reverted.

First, because changing `build-rootfs.sh` alone does not make sense until `eng/common/cross/toolchain.cmake` and (optionally) `eng/native/tryrun.cmake` also get the necessary changes for this particular coss-compilation setup.

Secondly, `eng/common` is a mirror directory, changes (when needed) should go in https://github.com/dotnet/arcade/tree/main/eng/common and the bot will mirror it (otherwise these changes will be overwritten with the next update).

Reply:
Changes under eng/common/cross/toolchain.cmake and eng/native/tryrun.cmake has been already done. 
Yes agree this change should go under arcade repository. Will do that.